package.json

```diff
@@ -0,0 +1,24 @@
+{
+  "name": "website-dashboard",
+  "version": "1.0.0",
+  "description": "Dashboard that performs authorized actions acrosss the website",
+  "main": "script.js",
+  "scripts": {
+    "start": "npx prettier --write",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
```

Comment:
Please make this package.json similar to this one
![image](https://user-images.githubusercontent.com/56217868/123459218-9e0f9780-d603-11eb-9d0b-95fcff197655.png)

Reply:
except the part after line 13